.github/workflows/pr-build-image.yaml

```diff
@@ -76,6 +68,7 @@ jobs:
           # to autodelete PR image tags, set an expiry date
           imageLabels: quay.expires-after=14d
           push: true
+          platform: linux/amd64
```

Comment:
so PR builds are single-arch but next builds are dual-arch? 🤔 What if your contributor is on a Mac and wants to test their PR build?

Reply:
Yes, for now yes.

My plan is to do the PR images in followup PR.
As it will require small updates to e2e tests as well.
e2e tests should **not** wait for the whole multiplatform image to be built, as it can take 4+ hours. So we will need to reconfigure them to wait for the platform specific tag.